src/util/brands-url.ts

```diff
@@ -19,8 +19,8 @@ export const brandsUrl = (options: BrandsOptions): string =>
 
 export const hardwareBrandsUrl = (options: HardwareBrandsOptions): string =>
   `https://brands.home-assistant.io/hardware/${options.category}/${
-    options.darkOptimized ? "dark_" : ""
-  }${options.manufacturer}${options.model ? `_${options.model}` : ""}.png`;
```

Comment:
Should we? Or should I fix a fallback on the brands repository instead?
That way, we could provide dark images if we want?

Reply:
Yea if we can fix it in brands that may be better. Not sure if we will have dark images for the hardware but we could

Reply:
Alright drafting and assigning myself. Going for some sleep first and will look at it in the morning.